### PR TITLE
Admin interview test progress

### PR DESCRIPTION
**Pull request by nonprofittechy**
* Wire up filing code endpoints; 
* add update notification preferences button.
* attach/detach service contact on existing case
* Update service contact

get_filings works, and added picklist for filing_id that works. however: get_filing_status appears to return meaningless information. see_filing_info, cancel_filing, and check_filing have various errors still.

### CODE REVIEW

File: docassemble/EFSPIntegration/efm_client.py [hunk 1]
```diff
@@ -361,17 +362,14 @@ def get_service_contact(self, service_contact_id):
...
-  def update_service_contact(self, service_contact_id, **kwargs):
+  def update_service_contact(self, service_contact_id, service_contact:Individual):
+    service_contact_dict = serialize_person(service_contact)
     send = lambda: self.proxy_client.patch(self.base_url + f'firmattorneyservice/service-contacts/{service_contact_id}', 
-        data=json.dumps(kwargs))
+        data=json.dumps(service_contact_dict))
     return self._call_proxy(send)
 
-  def create_service_contact(self, service_contact:Individual, is_public:bool, is_in_master_list:bool, admin_copy:str=None):
+  def create_service_contact(self, service_contact:Individual):
```
Comment (BryceStevenWilley) [inline docassemble/EFSPIntegration/efm_client.py:371]: We still need these extra parameters when creating a service. Was this just a merge issue, or did you intentionally remove them?

File: docassemble/EFSPIntegration/efm_client.py [hunk 2]
```diff
@@ -527,15 +530,15 @@ def serialize_person(person:Union[Person,Individual])->Dict:
...
     return_dict = {
-      "firmName": person.name.text if hasattr(person.name, 'text') else None,
+      "firmName": person.name.text,
     }
   return_dict.update({
     "address": {
-      "addressLine1": person.address.address if hasattr(person.address, 'address') else None,
+      "addressLine1": person.address.address,
       "addressLine2": person.address.unit if hasattr(person.address, 'unit') else None,
-      "city": person.address.city if hasattr(person.address, 'city') else None,
```
Comment (BryceStevenWilley) [inline docassemble/EFSPIntegration/efm_client.py:536]: Same with these as well, it looks like a merge issue (or just a Docassemble github issue)? There are time where we don't need to provide the address. We can still always ask in the yml, but it shouldn't be forced here.

**Review by BryceStevenWilley — CHANGES_REQUESTED** [review]
Seems like there's some merge issues here. I can help rebase if you'd like, just let me know.

File: docassemble/EFSPIntegration/efm_client.py (hunk 1 above)
Comment (nonprofittechy) [reply docassemble/EFSPIntegration/efm_client.py:371]: not something I remember touching so probably a merge issue--no objection to restoring those. interestingly it worked without but i will add those parameters as questions in the interview

File: docassemble/EFSPIntegration/efm_client.py (hunk 2 above)
Comment (nonprofittechy) [reply docassemble/EFSPIntegration/efm_client.py:536]: I'll fix this one too

**Review by BryceStevenWilley — APPROVED** [review]

**Pull request merged**